sphinx_source/standart_moduller/json.rst

```diff
@@ -0,0 +1,215 @@
+    `dump` fonksiyonu çıktıyı illaki bir dosya içine aktarır.
+    Yani size al bu senin istediğin JSON çıktısı demez. Bunu diyen
+    `dumps` fonksiyonudur. `dumps` fonksiyonu `str` tipinde
+    bir değer döndürürken `dump` fonksiyonu hiçbir değer döndürmez.
+
+O yüzden python.org sitesinden alınan şu örneklerde `dump`
+yerine `dumps` fonksiyonu kullanılmış.Eğer `dump` fonksiyonunu
+kullanacaksanız JSON'a dönüşmesini istediğiniz ifadeden sonra
+dosyayı bulunduran değişkeni yazın.
+::
+
+    >>> import json
+    >>> json.dumps(['foo', {'bar': ('baz', None, 1.0, 2)}])
+    '["foo", {"bar": ["baz", null, 1.0, 2]}]'
+    >>> print(json.dumps("\"foo\bar"))
+    "\"foo\bar"
+    >>> print(json.dumps('\u1234'))
+    "\u1234"
+    >>> print(json.dumps('\\'))
+    "\\"
+    >>> print(json.dumps({"c": 0, "b": 0, "a": 0}, sort_keys=True))
+    {"a": 0, "b": 0, "c": 0}
+    >>> from io import StringIO
+    >>> io = StringIO()
+    >>> json.dump(['streaming API'], io)
+    >>> io.getvalue()
+    '["streaming API"]'
+
+Bu fonksiyonların birkaç parametresi var. Şimdi sıra bunların
+ne işe yaradığını öğrenmekte.
+
+**skipkeys**
+
+Normalde Python, JSON oluştururken anahtar veya değer basit tipte
+(str,int,float...) değilse `TypeError` hatası verir.
+Eğer bu parametre `True` (Varsayılan `False`) ise hata vermek yerine o ikiliyi atlar.
+
+**ensure_ascii**
+
+Eğer bu parametre `True` (varsayılan olarak) ise çıktıda ASCII
+tablosuna uymayan karakterlerden kaçınır. `False` ise buna dikkat etmez.
+
+::
+
+    >>> import json
+    >>> json.dumps(["üzüm"],ensure_ascii=True)
+    '["\\u00fcz\\u00fcm"]'
+    >>> json.dumps(["üzüm"],ensure_ascii=False)
+    '["üzüm"]'
+
+**indent**
+
+Eğer negatif olmayan bir tamsayı veya bir karakter dizisi ise
+girintileme sayesinde daha güzel bir çıktı almanızı sağlar.
+Eğer 0, negatif sayı veya boş karakter dizisi(`""`) ise her öğeyi yeni 
+satıra basar. None(varsayılan) ise dip dibe bir çıktı verir.
+Pozitif bir sayı verildiğinde onu boşluk sayısı kabul ederek 
+girintileme yapar. Karakter dizisi ifadesine şu ikisi yeterince açık
+bir örnek: `\t`, `\n`
+
+.. Note:: Karakter dizisi kullanımı 3.2 sürümü itibari ile başladı.
+
+::
+
+    >>> print(json.dumps({"Özellikler":{"Hız":150,"Ses":"10db"}},indent=None))
+    {"\u00d6zellikler": {"H\u0131z": 150, "Ses": "10db"}}
+    >>> print(json.dumps({"Ozellikler":{"Hız":150,"Ses":"10db"}},indent=4))
+    {
+        "Ozellikler": {
+            "Hiz": 150,
+            "Ses": "10db"
+        }
+    }
+    >>> print(json.dumps({"Ozellikler":{"Hız":150,"Ses":"10db"}},indent="\n"))
+    {
+
+    "Ozellikler": {
+
+
+    "Hiz": 150,
+
+
+    "Ses": "10db"
+
+    }
+    }
+    >>> print(json.dumps({"Ozellikler":{"Hız":150,"Ses":"10db"}},indent=0))
+    {
+    "Ozellikler": {
+    "Hiz": 150,
+    "Ses": "10db"
+    }
+    }
+
+**separators**
+
+Bu parametre verilen değeri öğeler arasında ayraç olarak 
+kullanır. Verilen değer tuple tipinde olmalıdır. Varsayılan olarak
+şu kullanılır: `(",",": ")`
```

Comment:
Bunun anlamı nedir: `(",",": ")`

Reply:
O separators parametresinin varsayılan değeri

Reply:
Anladım. Ayraç olan virgülden sonra boşluk olmadığı için karıştırmışım. Buraya bir boşluk koyar mısınız? Şu şekilde: (",", ": ")
Bir de şuradaki resmi belgeye göre `indent` parametresinin değerine göre bu parametrenin varsayılan değeri değişiyormuş: https://docs.python.org/3/library/json.html Bu kısmı buna uygun olarak güncelleyebilir misiniz?

Reply:
Ekledim.